pymatgen/ext/matproj.py

```diff
@@ -492,7 +493,12 @@ def get_pourbaix_entries(self, chemsys):
         ion_ref_entries = self.get_entries_in_chemsys(
             list(set([str(e) for e in ion_ref_elts] + ['O', 'H'])),
             property_data=['e_above_hull'], compatible_only=False)
-        compat = MaterialsProjectAqueousCompatibility("Advanced")
+
```

Comment:
I suspect we need to add this logic to `DbPourbaix`.

Reply:
Actually, I think the quickest fix is to pass `solid_compat=MaterialsProjectCompatibility()` as an arg to `MaterialsProjectAqueousCompatibility` when calling `DbPourbaix`. I'll push that change

Reply:
https://github.com/materialsproject/pymatpro/commit/e8577128a42d6b780c7a3b7aaac4b81367fc05f0

Reply:
@tschaume let me know if my last commit fixes the issues. There's a lot of duplicated code between pymatgen, pymatpro, and django re: construction of Pourbaix Diagrams, so I think this was just a case in which we updated MPRester but didn't duplicate the updates in pymatpro. I'll look into consolidating this so that future releases won't have these issues like these.

Reply:
@rkingsbury thank you! Unfortunately, the Fe-Cr Pourbaix Diagram still looks off:

<img width="424" alt="Materials_Project" src="https://user-images.githubusercontent.com/1404564/88498898-44c56100-cf79-11ea-9ef1-56fe52f91e48.png">

Reply:
@tschaume can you try again with the latest? I think there was a problem with how `DbPourbaix` was constructing a temporary entry

Reply:
@rkingsbury yay! This looks sensible :) Only had to fix the `ComputedEntry` import from `pymatgen.entries.computed_entries`. I'll build a new image and we'll try to deploy again. I'll get back to you if any other issues pop up. Thanks!!

<img width="424" alt="Materials_Project" src="https://user-images.githubusercontent.com/1404564/88501844-7c380b80-cf81-11ea-9b59-089abeeb9a12.png">

Reply:
Thanks both for addressing this + fixing so quickly!